EIPS/eip-3475.md

```diff
@@ -1,17 +1,16 @@
 ---
 eip: 3475
-title: Multiple Callable Bonds
-author: Yohji Sakamoto (@sgmfinance)
-discussions-to: https://github.com/ethereum/EIPs/issues/3467
+title: ERC-3475 Multiple Callable Bonds Standard
+description: A standard interface for contract that manage multiple bonds. This standard will allow multiple LP token contract to store more information
+author: Yu LIU (@yuliu-debond), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold (@Edoumou), Toufic Batrice (@toufic0710)
+discussions-to: https://github.com/ethereum/EIPs/pull/4858/
```

Comment:
This is not an appropriate `discussions-to` link, please open a thread on ethereum magicians forum.